package.json

```diff
@@ -11,7 +11,7 @@
     "deploy-package-production": "node-lambda deploy -e production -f deploy_production.env",
     "build-doc-deploy-package-qa": "npm run build-swagger-doc && npm run deploy-package-qa",
     "build-doc-deploy-package-production": "npm run build-swagger-doc && npm run deploy-package-production",
-    "test": "swagger project test"
+    "test": "jest"
```

Comment:
Is `jest` the correct command or is it a typo for `test`?

Reply:
Thanks for checking, @EdwinGuzman.  I'm using Facebook's Jest for JS testing: http://jestjs.io/ so although it looks like "jest" could be a typo, it is correct.